src/annotator/highlighter.js

```diff
@@ -210,19 +237,19 @@ export function highlightRange(normedRange, cssClass = 'hypothesis-highlight') {
  *
  * nb. This is like `ChildNode.replaceWith` but it works in IE 11.
  *
- * @param {Node} node
+ * @param {ChildNode} node
```

Comment:
The `remove()` method used in this function only exists on `ChildNode`-y nodes (eg. `Element`s)

Reply:
👍